lib/css_to_emotion.re

```diff
@@ -5,101 +5,9 @@ open Asttypes;
 open Parsetree;
 open Css_types;
 
-type mode =
-  | Bs_css
-  | Bs_typed_css;
-
 let grammar_error = (loc, message) =>
   raise([@implicit_arity] Css_lexer.GrammarError(message, loc));
 
-let is_overloaded = (mode, declaration) =>
-  /* Overloaded declarations are rendered as function applications where function name
-        ends with a number that specifies the number of parameters.
-        E.g.: margin: 1em 2px -> margin2 em(1.) px(2.)
-     */
-  switch (mode) {
-  | Bs_css => false
-  | Bs_typed_css =>
-    switch (declaration) {
-    | "unsafe" => false
-    | _ => true
-    }
-  };
-
-let is_variant = (mode, ident) =>
-  /* bs-css uses polymorphic variants to enumerate values. Some of them
```

Comment:
`is_variant` or something similar might be needed for bs-emotion as well, as many values are variants, e.g. https://github.com/ahrefs/bs-emotion/blob/master/bs-emotion/src/Emotion.rei#L681